atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -420,6 +421,12 @@ def get_build(self, metadata, worker_metadatas):
         if flatpak_source_info is not None:
             extra['image'].update(flatpak_source_info.koji_metadata())
 
+        if koji_task_id:
+            koji_task_owner = get_koji_task_owner(self.session, koji_task_id, default=None)['name']
+        else:
+            koji_task_owner = None
+        extra['submitter'] = self.session.getLoggedInUser()['name']
```

Comment:
I don't think its correct - `getLoggedInUser` most likely would return koji account OSBS uses (`osbs`), not the original submitter.

See `_get_koji_submitter` in `exit_sendmail.py`

Reply:
Actually, submitter is the one who submitted the build via content generator API (osbs), and owner is the user that started the task (me, for example). The owner is used to populate the "Built by" field in the UI, for example. The submitter field is simply added for tracking purposes, AFAIK it's not actually actively used.

@breillyr, please confirm.

Reply:
Ah! Yes, now I see. "owner" is described in the [documentation](https://docs.pagure.org/koji/content_generator_metadata/).

I don't see "submitter" described anywhere except for the description of [this issue](https://pagure.io/koji/issue/299#), but from the story description the code looks right.